Adding ampliseq improvements project (#3112)
nf-core/website: sites/main-site/src/content/hackathon-projects/hackathon-march-2025/ampliseq-improvements.md

---
title: Adding, Moding, Menddling with the nf-core/ampliseq Pipeline
category: pipelines

> slack: ""

Reviewer: ```suggestion
```

intro_video: ""
image: "/assets/images/events/2025/hackathon-march/meddling_with_ampliseq.jpg"

> image_alt: Meddling with ampliseq

Reviewer: please write a real alt-text https://accessibility.huit.harvard.edu/describe-content-images

leaders:
  npechl:
    name: Nikos Pechlivanis
    slack: "https://nfcore.slack.com/team/U083L3UDXMG"
  Sofokli5:
    name: Sofoklis Keisaris
    slack: "https://nfcore.slack.com/team/D08CT7DSN20"
---
This project aims to enhance the [nf-core/ampliseq](https://nf-co.re/ampliseq/dev) pipeline to support plant metabarcoding analyses. By expanding the reference database, we will improve its functionality, fostering broader adoption and making plant metabarcoding more accessible to the scientific community.

## Goals
- Expanding the reference sequences for plant metabarcoding analyses.
- Integrating the new references following the latest nf-core and Nextflow standards.  
- Benchmarking and validating the pipeline to ensure error-free performance.  

> - Improving documentation and user support to promote broader adoption.

Reviewer: please mark the experience level of the different goals

---

> title: Adding, Moding, Menddling with the nf-core/ampliseq Pipeline

Reviewer: ```suggestion
title: Adding, modding, meddling with the nf-core/ampliseq pipeline
```

category: pipelines
intro_video: ""
image: "/assets/images/events/2025/hackathon-march/meddling_with_ampliseq.jpg"

> image_alt: "Distracted boyfriend" meme for ampliseq pipeline improvements

Reviewer: ```suggestion
image_alt: "'Distracted boyfriend' meme, where the boyfriend in the middle is labeled 'us', they oblivious woman to the right is labeled 'leaving ampliseq alone and fine as it is'. The woman on the left, who the boyfriend is whistling after, is labeled 'meddle with it to suppoert metabarcoding'."
```

leaders:
  npechl:
    name: Nikos Pechlivanis
    slack: "https://nfcore.slack.com/team/U083L3UDXMG"
  Sofokli5:
    name: Sofoklis Keisaris
    slack: "https://nfcore.slack.com/team/D08CT7DSN20"
---

> This project aims to enhance the [nf-core/ampliseq](https://nf-co.re/ampliseq/dev) pipeline to support plant metabarcoding analyses. By expanding the reference database, we will improve its functionality, fostering broader adoption and making plant metabarcoding more accessible to the scientific community.

Reviewer: ```suggestion
This project aims to enhance the [nf-core/ampliseq](https://nf-co.re/ampliseq/dev) pipeline to support plant metabarcoding analyses. 
By expanding the reference database, we will improve its functionality, fostering broader adoption and making plant metabarcoding more accessible to the scientific community.
```
